plugin/pkg/scheduler/algorithm/BUILD

```diff
@@ -21,6 +21,7 @@ go_library(
         "//pkg/api/v1:go_default_library",
         "//pkg/apis/extensions/v1beta1:go_default_library",
         "//plugin/pkg/scheduler/api:go_default_library",
+        "//vendor:k8s.io/client-go/pkg/apis/apps",
```

Comment:
A question here: Why not use `apps "k8s.io/kubernetes/pkg/apis/apps/v1beta1"` here? I saw that many other places are using `apps "k8s.io/kubernetes/pkg/apis/apps/v1beta1"` for `StatefulSet`.